### DISCUSSION TITLE
[rocket::config::error][ERROR] I/O error while setting tls.certs even when /ssl lists cert folders inside of container

### DISCUSSION THREAD

**Discussion by tangowithfoxtrot** [dani-garcia/vaultwarden#1259] [General]
I cannot replace my official Bitwarden implementation with bitwarden_rs. I followed Bitwarden's installation guide and have a working setup with SSL through Letsencrypt. All of my configuration data is stored within /home/user/bwdata.

`docker run -d --name bitwarden-rust -e ROCKET_TLS='{certs="/home/user/bwdata/letsencrypt/live/bw.domain.com/fullchain.pem",key="/home/user/bwdata/letsencrypt/live/bw.domain.com/privkey.pem"}' -v /home/user/bwdata/letsencrypt/:/ssl/ -v /home/user/bwdata/:/data/ -p 443:80 bitwardenrs/server:latest`

This command immediately closes with the `[rocket::config::error][ERROR] I/O error while setting tls.certs:` error. If I connect to a terminal while it executes, running an ls on the /ssl folder lists my accounts, archive, csr, keys, live, logs, renewal, and renewal-hooks folder. Is this the correct mapping for the container? I've tried mapping alternative paths that are higher up or lower in the letsencrypt folder, but with no luck.

**Comment by Zeldri** [comment]
Same here 🤔 #1260

**Comment by BlackDex** [comment]
You mount the lets encrypt folder within the container to /ssl/ yet you configure the ssl certs as if they are on the host.
You should either use `/data/letsencrypt/live` or `/ssl/live` for the `ROCKET_TLS` variable.
